src/getSongLink/getSongLink.ts

```diff
@@ -40,41 +40,7 @@ export const getIdFromEntity = (entity: string): string => {
 
 export const getDetailedMetadata = async (data: SonglinkResponse): Promise<MetaData> => {
 	if (data.linksByPlatform.deezer) {
-		try {
```

Comment:
This has been moved: https://github.com/Puffycheeses/music_linkr/pull/9/files#diff-1a34a9ba5dc35a5388ebf9fd791c907aaee81b3a04efc2bc76b51eb6a8cebcd1